src/RegisterHandler.php

```diff
@@ -117,18 +120,19 @@ public function register(HTTPRequest $request, StoreInterface $store): Result
             $response = $publicKeyCredential->getResponse();
 
             if (!$response instanceof AuthenticatorAttestationResponse) {
-                die('why even have this?');
+                throw new RuntimeException('Unexpected response type found');
```

Comment:
Just an idea, you may want to subclass RuntimeException in this module to provide a little more specificity for clients catching it